tensorboard/pip_package/BUILD

```diff
@@ -20,4 +23,54 @@ sh_binary(
         "//tensorboard/plugins/beholder",  # User-facing beholder API
         "//tensorboard/plugins/projector",  # User-facing projector API
     ],
+    tags = [
+        "local",
+        "manual",
+    ],
+)
+
+genrule(
+    name = "license",
```

Comment:
Please get my approval when adding new dependencies. That way I can make sure we do things like this. There's also other burdens, such as ensuring `/** @license ... */` comments appear in HTTP responses for works with notice licenses, which is oftentimes difficult since tooling might remove those comments, or the copyright holders might not have included them in the sources. It's a tough problem.